homework-submission/my-project/src/Description.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class Description extends Component {
+  render() {
+    return (
+      <div>
+        <p>{this.props.description}</p>
+      </div>
```

Comment:
it is work fine here but you don't need a < div > container because you are returning just one element you need a container just if you return more than one element

Reply:
all done!

Reply:
Well done Rim nice work

Reply:
Thanks!